src/kolibri_android/referrer.py

```diff
@@ -0,0 +1,93 @@
+import logging
```

Comment:
Nitpick: Copyright and licensing header missing, if that’s something we’re caring about at the moment.

I [can’t actually work out](https://github.com/learningequality/kolibri-installer-android/issues/141) what license the project is under, though, so I guess we should miss off the license statement for now. Adding the copyright now means we don’t need to do archaeology on that later, though.

Suggested:
```py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
# Copyright 2023 Endless OS Foundation, LLC
```

Reply:
Indeed, I started writing something and then punted. This is ostensibly a fork of LE's kolibri-installer-android and they license their stuff MIT. So, we could presumably license it however we want on top of that. However, I suspect we want make it MIT to maximize sharing back to LE. But we probably need confirmation of that first.

Reply:
I opened #137 to handle the licensing and will be happy to add headers to our code when it's clear what the license is.